src/python/pants/help/help_info_extracter.py

```diff
@@ -322,7 +323,7 @@ class PluginAPITypeInfo:
     name: str
     module: str
     documentation: str | None
-    provider: str
+    provider: tuple[str, ...]
```

Comment:
This change is not backwards compatible, which is a nuance. Probably not used extensively by end users, but we consume it for docs generation, so need to adapt there.

Reply:
Just noticed you have a checked box for this affecting docs.. so 🤷🏽 guess I'm off base here 😬